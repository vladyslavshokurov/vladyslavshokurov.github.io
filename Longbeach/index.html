<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Long Beach Surfing Institute</title>
	<link rel="stylesheet" type="text/css" href="style.css">
	<link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,900|Playfair+Display:400,400i" rel="stylesheet"> 
</head>
<body>
	<div class="nav-wrapper">
		<div class="logo">
			<a href="#">
				<img src="img/logo/Logo.svg" alt="Long Beach Surfing Institute">
			</a>
		</div>
		<div class="nav">
			
				<a href="#" class="nav__item nav__item--active">home</a>
				<a href="#" class="nav__item">about</a>
				<a href="#" class="nav__item">blog</a>
				<a href="#" class="nav__item">contact</a>
			
		</div>
	</div>
	<header>
		<h1 class="header-title">Long Beach Surfing Institute	
		</h1>
		<p class="header-description">
					Lorem ipsum dolor sit amet, consectetur adipisicing 		elit, sed do eiusmod tempor<br>  incididunt ut labore et 	dolore magna aliqua. Ut enim ad minim veniam.
		</p>
	<div class="header-content">	
		<div class="rewiews">
			<div class="rewiews-title">Recent Reviews
			</div>
			<div class="rewiew-item">
				<div class="rewiew-text">Lorem ipsum dolor sit amet, 			consectetur adipisicing elit, sed incididunt ut labore 		et dolore magn.
				</div>
				<div class="rewiew-author"> &#160; Steve Woz
				</div>
			</div>
			<div class="rewiews-title">Recent Reviews
			</div>
			<div class="rewiew-item">
				<div class="rewiew-text">Lorem ipsum dolor sit amet, 			consectetur adipisicing elit, sed incididunt ut labore 		et dolore magn.
				</div>
				<div class="rewiew-author"> &#160; David Collum
				</div>
			</div>
		</div>
	 	<div class="video-centre">
	 		<div class="play-icon">
			</div>
	 		<img src="img/video/img-1.jpg">
	 	</div>


		<div class="video">
			<div class="video-title">User Videos
			</div>
			
			<div class="video-item">
				<div class="play-icon">
				</div>
				<img src="img/video/img-2.jpg">
			</div>
			<div class="video-item">
				<div class="play-icon">
				</div>
				<img src="img/video/img-3.jpg">
			</div>
		</div>
	</div>		
	</header>
	<div class="container">
		<h2 class="instructors-title">Popular Instructors
		</h2>
			<div class="instructors">
				<div class="instructors-item">
					<img src="img/instructors/img-4.jpg">
					<div class="instructors-names">
						Simon Adward
					</div>
					<div class="instructors-description">
						5,000 students
					</div>
				</div>
				<div class="instructors-item">
					<img src="img/instructors/img-5.jpg">
					<div class="instructors-names">
						Elliana D’Soza
					</div>
					<div class="instructors-description">
						4,350 students
					</div>
				</div>
				<div class="instructors-item">
					<img src="img/instructors/img-6.jpg">
					<div class="instructors-names">
						Joseph Bardon
					</div>
					<div class="instructors-description">
						3,248 students
					</div>
				</div>
			</div>
		<h2 class="blogs-title">Recent Blogs by User
		</h2>	
			<div class="blogs">
				<div class="blogs-item1">
					<div class="blogs-image"><img src="img/blog/img-7.jpg"></div>
					<div class="blogs-item__text">
						<div class="blogs-item__text_sub-article">
							Lipsum&#160;cound&#160;be&#160;headline 
						</div>
						<div class="blogs-item__text_description">
							Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.
						</div>
						<div class="blogs-button"><a href="#">Read More</a></div>
					</div>

				</div>
				<div class="blogs-item2">
					<div class="blogs-image"><img src="img/blog/img-8.jpg"></div>
					<div class="blogs-item__text">
						<div class="blogs-item__text_sub-article">
							Lipsum&#160;cound&#160;be&#160;headline 
						</div>
						<div class="blogs-item__text_description">
							Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.
						</div>
						<div class="blogs-button"><a href="#">Read More</a></div>
					</div>
				</div>
			</div>
			<h2 class="advertising">Aren’t you intrested Yet!!</h2>
			<div class="advertising-button"><a href="#">Enroll Now</a></div>
	</div>
	<footer class="footer">
		<div class="footer-container">
			<div class="footer-copyright">
				<div class="footer-copyright__title">Beach Surfing</div>
				<div class="footer-copyright__text">Copyright © 2018 All Rights Reserved.</div>
			</div>
			<div class="footer-designedby"><span class="footer-designedby-text__designedby">Designed by</span><span class="footer-designedby-text__name"> Rikon Rahman</span></div>
		</div>
	</footer>

	

 

		




</body>
</html>