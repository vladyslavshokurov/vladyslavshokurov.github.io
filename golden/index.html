<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" type="text/css" href="style.css">
</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img src="images/logo.png" alt="logo" class="logo">
				<nav>
					<ul class="menu">
						<li><a href="#">Home<a></li>
						<li><a href="#">Services<a></li>
						<li><a href="#">Portfolio<a></li>
						<li><a href="#">About<a></li>
						<li><a href="#">Contact<a></li>
					</ul>
				</nav>
			</div>
		</div>	
		<div class="titles">
			<div class="titles__first">
				<p>Welcome to our studio!</p>
			</div>
			<h1>
				It's nice to meet you
			</h1>
		</div>
		<a href="#" class="button">Tell me more</a>
	</header>
	<section class="container">
		<div>
			<div class="title">
				<h2>
					Services
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>
			</div>
		</div>	
		<div class="services clearfix">
			<div class="services__item">
				<img src="images/icon1.png">
				<h3>E-Commerce</h3>
				<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.
				</p>
			</div>
			<div class="services__item">
				<img src="images/icon2.png">
				<h3>Responsive Web</h3>
				<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.
				</p>
			</div>
			<div class="services__item">
				<img src="images/icon3.png">
				<h3>Web Security</h3>
				<p>Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt.
				</p>
			</div>
		</div>
	</section>
	<section id="portfolio">
		<div  class="container">
			<div class="title">
				<h2>
					our portfolio
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>
			</div>
			<div class="works clearfix">
				<a href="#"><img src="images/img1.jpg"></a>
				<a href="#"><img src="images/img2.jpg"></a>
				<a href="#"><img src="images/img3.jpg"></a>
			</div>
		</div>	
	</section>
	<section class="container">
		<div>
			<div class="title">
				<h2>
					about us
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>
			</div>
			<div class="about">	
					<div class="about__item flex-item1">
						<div class="flex-text1">
							<h3><span>July 2010</span><br>
								Our Humble Beginnings
							</h3>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit.
							</p>
						</div>
						<div class="flex-img1">
							<img src="images/about1.jpg">
						</div>
						<div class="last">
						</div>
					</div>
			</div>	
			<div class="about">	
					<div class="about__item flex-item2">
						<div class="flex-text2">
							<h3><span>January 2011</span><br>
								Facing Startup Battles
							</h3>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit aenean 
								rhoncus posuere odio in tincidunt.
							</p>
						</div>
						<div class="flex-img2">
							<img src="images/about2.jpg">
						</div>
						<div class="last2">
						</div>
					</div>
			</div>	
			<div class="about">	
					<div class="about__item flex-item3">
						<div class="flex-text3">
							<h3><span>December 2012</span><br>
								Enter The Dark Days
							</h3>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni.
							</p>
						</div>
						<div class="flex-img3">
							<img src="images/about3.jpg">
						</div>
						<div class="last">
						</div>
					</div>
			</div>	
			<div class="about">	
					<div class="about__item flex-item4">
						<div class="flex-text4">
							<h3><span>february 2014</span><br>
									Our Triumph
							</h3>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit aenean.
							</p>
						</div>
						<div class="flex-img4">
							<img src="images/about4.jpg">
						</div>
						<div class="last4">
						</div>
					</div>
			</div>		
		</div>	
	</section>
	<footer class="container">
		<p>Â© Copyright 2014 FreebiesXpress.com</p>
		<div>

		</div>	
	</footer>


	
</body>
</html>